### DISCUSSION TITLE
Kashmiri Language Dictionary

### DISCUSSION THREAD

**Discussion by MAJIDNISAR** [github/india#79] [Open Source] [answered]
I want to create An API for kashmiri language translation and transliteration,
what should be the next steps, please suggest.

**Comment by knadh** [comment, marked as answer]
You could start by studying the [building blocks](https://github.com/github/india/discussions/74#discussioncomment-536972) to get a sense on how to build a local language dictionary.

**Reply by MAJIDNISAR** [reply]
let me get started, can you buzz on social if i need help and ask you to contribute @knadh
